art/estimators/classification/tensorflow.py

```diff
@@ -910,6 +910,34 @@ def class_gradient(self, x, label=None, **kwargs):
 
         return gradients
 
+    def loss_gradient_framework(self, x, y, **kwargs):
+        """
+        Compute the gradient of the loss function w.r.t. `x`.
+
+        :param x: Input with shape as expected by the model.
+        :type x: `Tensor`
+        :param y: Target values (class labels) one-hot-encoded of shape (nb_samples, nb_classes) or indices of shape
+                  (nb_samples,).
+        :type y: `np.ndarray`
+        :return: Gradients of the same shape as `x`.
+        :rtype: `Tensor`
+        """
+        import tensorflow as tf
+
+        if tf.executing_eagerly():
```

Comment:
What happens if TensorFlow is not executing eagerly? Should we raise an exception like NotImplementedError?

Reply:
Done.